### DISCUSSION TITLE
MAISI：VAE & Diffusion Unet model weight file

### DISCUSSION THREAD

**Discussion by boxiangyun** [Project-MONAI/tutorials#1903] [Q&A] [answered]
Thank you for releasing the detailed MAISI training code. Will the VAE and Diffusion Unet network weight files be made open source in the future?

**Comment by KumoLiu** [comment, marked as answer]
Hi @boxiangyun, you can find all the pre-trained weights in the model-zoo repo: https://github.com/Project-MONAI/model-zoo/blob/dev/models/maisi_ct_generative/large_files.yml

For more details about the config, you can also refer to the model folder: https://github.com/Project-MONAI/model-zoo/tree/dev/models/maisi_ct_generative

Hope it helps, thanks.
